### ISSUE TITLE
Documentation request for OMNet++ 6.0.1 / sequence chart / context menu / configure style

### ISSUE THREAD

**Issue by windy32** [omnetpp/omnetpp#953]
**Platform**
 - OMNeT++ version: 6.0.1
 - OS: Linux Mint 20.2 MATE 64 bit

**Describe the bug**
There is a screenshot in INET / Showcases / Time-Sensitive Networking / Scheduling and Traffic Shaping / Time-Aware Shaping
The url is https://inet.omnetpp.org/docs/showcases/tsn/trafficshaping/timeawareshaper/doc/index.html
The screenshot is
![screenshot4](https://user-images.githubusercontent.com/12342773/199187538-6b1239cd-484c-43e1-87e1-28557f7dcfea.png)

I am able to follow the sequence chart configuration, except the configuration of message styles.

![screenshot3](https://user-images.githubusercontent.com/12342773/199188935-36af4ef2-aec7-42f8-a239-29477c2fa73a.png)

I need to set the color of video messages to red. The "configure style" dialog needs an expression as the input, but I am unable to find  any related documentation.

Is there any documentation about sequence chart styling that I have missed?

**Comment by levy** [comment]
Not really, but basically you can write expressions with and/or and *, colors are either dark/light color groups or color names separated by commas. Colors are used cyclically.

![image](https://user-images.githubusercontent.com/112461/199580546-e492929f-41ef-435b-90e9-28a4a0a731d6.png)
